src/linter/html/linter.ts

```diff
@@ -12,9 +12,13 @@ export async function lintHtml(resourceName: string, contentStream: ReadStream):
 	const jsScriptTags = await extractJSScriptTags(contentStream);
 
 	jsScriptTags.forEach((tag) => {
-		const scriptContent = tag.textNodes?.map((tNode) => tNode.value).join("").trim();
+		const scriptContent = tag.textNodes?.map((tNode) => tNode.value).join("");
+		// Tags with src attribute do not parse and run inline code
+		const hasSrc = tag.attributes.some((attr) => {
+			return attr.name.value.toLowerCase() === "src";
+		});
 
-		if (scriptContent) {
+		if (!hasSrc && scriptContent) {
```

Comment:
Do you think it could be enough to check for the existence of any text node now?
If yes, we could remove the map/join calls above.

```suggestion
		if (tag.TextNodes?.length > 0 && scriptContent) {
```

Reply:
Hmm, thanks!

Reply:
Done with https://github.com/SAP/ui5-linter/pull/70/commits/4265bfb280f1ddf4ada8951727b41e6b1179fe84